src/wasm-interpreter.h

```diff
@@ -1343,7 +1343,7 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     return Flow(NONCONSTANT_FLOW);
   }
 
-  Flow visitCallIndirect(CallIndirect*) {
+  Flow visitCallIndirect(CallIndirect* curr) {
```

Comment:
I think we generally give unused arguments names all over the code base, so I think this should be part of a separate, larger change if we decide to do it.

Reply:
This was necessary to make the WASM_INTERPRETER_DEBUG work, since the macros use `curr`. Shall I revert it nonetheless?

Reply:
Oh interesting. Shows how often we enable `WASM_INTERPRETER_DEBUG`! Personally I would rather just remove `WASM_INTERPRETER_DEBUG`. Did you find it useful enough to keep around? @kripken what do you think?

Reply:
I found it quite useful, yeah. Pointed me at the expression causing the problem, while showing the code path it took to get there. Appeared, however, that `WasmPrinter` isn't actually printing the expressions (just random addresses), since it only has a `Expression&` overload, but none for `Expression*`. Monkey-patching that made it much more verbose, but also easier to understand :)

Reply:
Ok, in that case this seems fine, and it would probably be good to print the expressions properly as well since I agree that the random addresses are not great.

Reply:
Oh wait I think I got confused and read this backwards initially. Adding `curr` everywhere is exactly what I want anyway 👍